set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetServiceImpl.java

```diff
@@ -496,7 +497,13 @@ private Query<PersistentUserSet> buildUserConditionsQuery(UserSetQuery query) {
   private void buildSortCriteria(UserSetQuery query, Query<PersistentUserSet> mongoQuery) {
     for (String sortField : query.getSortCriteria()) {
       if (!sortField.contains(" ")) {
-        mongoQuery.order(Sort.ascending(sortField));
+        //check the text search score
```

Comment:
this is the case when no order is provided through the query. The sort by text score descending needs to be added